### DISCUSSION TITLE
Monal Chats order.

### DISCUSSION THREAD

**Discussion by izanp** [tigase/siskin-im#149] [Ideas]
Hi everyone, hope you are well.
Is it possible to have a "sort" function in order to sort your Chats history...for instance by last message sent/received/modified etc....

**Comment by hantu85** [comment]
The list of chats is already sorted by the time of last received/modified item (at least in version in TestFlight).

I do think that this is expected and the best way to sort list of chats. I'm not convinced that we should have any other way of sorting chats list, but what other options you would like to have? Stating same as Monal does not give me any idea.

**Reply by izanp** [reply]
Thank you for the reply Sir!
Ive noticed that mine just took a little long to register. It eventually did sort, just like you said.

Not sure why it took so long but it works fine like it is now..

Thanks
